android/src/main/java/com/heapanalytics/reactnative/RNHeap.java

```diff
@@ -0,0 +1,12 @@
+package com.heapanalytics.reactnative;
+
+import android.content.Context;
+import com.heapanalytics.android.Heap;
+import com.heapanalytics.android.internal.HeapImpl;
+
+public class RNHeap {
+    public static void init(Context context, String appId) {
+        HeapImpl.skipInstrumentorChecks = true;
```

Comment:
Also, is there somewhere else that we call this already from the RN Plugin?  It's totally idempotent to do it, but probably good to find any other spot where we do it already for hygiene sake.

Reply:
Yep, in `RNHeapLibraryModule.java`: https://github.com/heap/react-native-heap/blob/037e2e3cfed8d23ba7334dd2982870ab7c9619a9/android/src/main/java/com/heapanalytics/reactnative/RNHeapLibraryModule.java#L34

Not sure I understand the suggestion here.

Reply:
Just saying that we should only have the call in one spot.  It doesn't matter if we call it twice (idempotent), but, conceptually, this kind of initialization stuff should only happen once in our code.

Reply:
Nvm my last comment... I get it now :-P